### PR TITLE
PromCon 2023

### PR DESCRIPTION
**Pull request by metalmatze**

### CODE REVIEW

**Comment by SuperQ** [comment]
Want to push the Gemfile thing as a separate PR?

**Comment by metalmatze** [comment]
> Want to push the Gemfile thing as a separate PR?

I would, but then again it already is in here and seems to work just fine. Any other particular reason?

**Comment by SuperQ** [comment]
Just prefer to keep chore work separate from other changes.

File: content/2023-berlin/giggity.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "end": "2022-09-29",
+        "metadata": {
+            "icon": "https://promcon.io/assets/favicons/apple-touch-icon.png",
+            "links": [
+                {
+                    "url": "https://promcon.io/2023-berlin/",
+                    "title": "Website"
+                },
+                {
+                    "url": "",
```
Comment (SuperQ) [inline content/2023-berlin/giggity.json:15]:
```suggestion
                    "url": "https://goo.gl/maps/rV8d4XZAJczvu6Vr7",
```

**Comment by metalmatze** [comment]
Rebased and removed those changes.

File: content/2023-berlin/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: Overview
+---
+
+## Overview
+
+PromCon EU 2023 is the eigth conference fully dedicated to the
```
Comment (juliusv) [inline content/2023-berlin/index.md:7]:
```suggestion
PromCon EU 2023 is the eighth conference fully dedicated to the
```

Or just say "8th" :)

Comment (metalmatze) [reply content/2023-berlin/index.md]: Thanks! This seems more elevated :smiley:

**Comment by roidelapluie** [comment]
LGTM, but promcon is not yet in the travel funding form of CNCF

**Comment by juliusv** [comment]
Looks good in general. The health & safety page could maybe use a little more updating (travel restrictions are gone for a long time and nobody is wearing masks anymore anywhere), but oh well.

**Comment by metalmatze** [comment]
> LGTM, but promcon is not yet in the travel funding form of CNCF

Yeah, we're overall waiting for the final "go" with the signature of the venue from CNCF side. We (Frederic, Jah and I) decided to just go ahead after waiting another week, to publish this without disclosing the actual location, such that the CfP can finally open and potential sponsors know more too. 

There are more updates to follow in the next hours and days. 

> The health & safety page could maybe use a little more updating

Yes, we've just discussed this too. I think we want to recommend wearing a mask. I agree on updating the text on local guidlines, however, I thought it's still fine to keep it in as is. Happy to remove it, if deemed way too outdated.

**Pull request merged**